docs/make.jl

```diff
@@ -28,7 +28,8 @@ pages = Any["Getting Started"=>"index.md",
             "MLJ Cheatsheet" => "mlj_cheatsheet.md",
             "MLJ News"=>"NEWS.md",
             "FAQ" => "frequently_asked_questions.md",
-            "Julia BlogPost"=>"julia_blogpost.md"]
+            "Julia BlogPost"=>"julia_blogpost.md",
+            "Acceleration and parallelism"=>"acceleration.md"]
```

Comment:
Finicky request: Match the title and file name for consistency. So, rename "acceleration.md" to "acceleration_and_parallelism.md"